RecoLocalCalo/HcalRecProducers/src/HBHEPhase1Reconstructor.cc

```diff
@@ -483,7 +485,8 @@ void HBHEPhase1Reconstructor::processData(const Collection& coll,
             auto s(frame[ts]);
             const uint8_t adc = s.adc();
             const int capid = s.capid();
-            const double pedestal = calib.pedestal(capid);
+            //optionally increase pedestal mean to account for dark current + crosstalk
+            const double pedestal = calib.pedestal(capid) + (saveEffectivePedestal_ ? darkCurrent * 25. / (1. - lambda) : 0.);
```

Comment:
When this parameter is enabled, the effective pedestal value is saved in HBHEChannelInfo for use in subsequent reco algorithms. I think it's just semantics...